File: Lista 2 sala/Python/atividade05.py
# 5)- Ler 3 valores (A, B e C) e exibir os valores dispostos em ordem crescente.

A=int(input("Digite um valor A:"))
B=int(input("Digite um valor B:"))
C=int(input("Digite um valor C:"))

if A==B and B==C and C==A:
    print("Todos os valores são iguais")
elif B>=C>=A:
    print("O valor de B é maior, a ordem crescente é B,C,A:",B,C,A)
elif C>=B>=A:
    print("o valor de C é maior, a ordem crescente é C,B,A:",C,B,A)
elif A>=B>=C:
    print("O valor de A é maior a ordem crescente é A,B,C:",A,B,C)

File: Lista 2 sala/C/atividade05.c
#include <stdio.h>

int main() {
    int A, B, C;

    printf("Digite um valor A: ");
    scanf("%d", &A);

    printf("Digite um valor B: ");
    scanf("%d", &B);

    printf("Digite um valor C: ");
    scanf("%d", &C);

    if (A == B && B == C) {
        printf("Todos os valores sao iguais\n");
    } else if (B >= C && C >= A) {
        printf("O valor de B e maior, a ordem crescente e B, C, A: %d %d %d\n", B, C, A);
    } else if (C >= B && B >= A) {
        printf("O valor de C e maior, a ordem crescente e C, B, A: %d %d %d\n", C, B, A);
    } else if (A >= B && B >= C) {
        printf("O valor de A e maior, a ordem crescente e A, B, C: %d %d %d\n", A, B, C);
    }

    return 0;
}
